src/checkers/inference/solver/frontend/TwoQualifiersLattice.java

```diff
@@ -14,41 +14,13 @@
  */
 public class TwoQualifiersLattice extends Lattice {
```

Comment:
I am not sure. We don't have to keep this class now, but using this class may make the use of two-qualifier lattice more obvious.